src/meltano/core/schedule_service.py

```diff
@@ -45,8 +49,12 @@ def __init__(self, name: str):
         """
         self.name = name
 
+        reason = f"Schedule '{self.name}' does not exist"
+        instruction = "Use `meltano schedule add` to add a schedule"
+        super().__init__(reason, instruction)
```

Comment:
Why store `name` on the attribute?

I think it would be nicer if `reason` and `instruction` were passed in as keyword arguments.

This same suggestion applies to a few other decedents of `MeltanoError`.

Reply:
@WillDaSilva problem I thought of that it would require the dev to type a reason and instruction every time the exception is instantiated (if the exception may be raised in different places)

Reply:
@edgarrmondragon Sorry for not being clear. When I said

> I think it would be nicer if `reason` and `instruction` were passed in as keyword arguments.

I didn't mean to suggest changing the function signature. I meant calling `super().__init__` like this:

```python
super().__init__(
    reason="...",
    instruction="...",
)
```

It was just a minor style comment, since that way avoids the use of additional variables, and would break (as we desire) if the signature is changed in the future.

Reply:
@WillDaSilva Ah, that makes sense. Sorry for my confusion 😅. Just pushed the change.